src/main/resources/org/jenkins/plugins/lockableresources/actions/LockableResourcesRootAction/index.properties

```diff
@@ -63,4 +66,6 @@ btn.editNote=Note
 btn.editNote.detail=Edit resource note.
 #table labels
 labels.table.column.labels=Labels
-labels.table.column.free=Free resources
\ No newline at end of file
+labels.table.column.free=Free
+labels.table.column.assigned=Assigned resources
```

Comment:
@gounthar Can you  pls translate it to franc for me. Thx

Reply:
Done, thank you.